src/config.ts

```diff
@@ -49,6 +49,14 @@ export function envHead (): boolean {
 
 }
 
+export function envDisableExtra (): boolean {
+  const KEY = 'WECHATY_PUPPET_PUPPETEER_EXTRA_DISABLED'
```

Comment:
How about let's rename `WECHATY_PUPPET_PUPPETEER_EXTRA_DISABLED` to `WECHATY_PUPPET_PUPPETEER_DISABLE_EXTRA`?